egs2/TEMPLATE/sv1/setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+help_message=$(cat << EOF
+Usage: $0 <target-dir>
+EOF
+)
+
+
+if [ $# -ne 1 ]; then
+    log "${help_message}"
+    log "Error: 1 positional argument is required."
+    exit 2
+fi
+
+
+dir=$1
+mkdir -p "${dir}"
+
+if [ ! -d "${dir}"/../../TEMPLATE ]; then
+    log "Error: ${dir}/../../TEMPLATE should exist. You may specify wrong directory."
+    exit 1
+fi
+
+targets=""
+
+# Copy
+for f in cmd.sh conf local; do
+    target="${dir}"/../../TEMPLATE/sv1/"${f}"
+    cp -r "${target}" "${dir}"
+    targets+="${dir}/${target} "
+done
+
+
+# Symlinks to TEMPLATE/diar1
```

Comment:
```suggestion
# Symlinks to TEMPLATE/sv1
```

Reply:
Thanks!! I already made this change locally.. Let me check what happened!

Reply:
I did this in [f83d811](https://github.com/espnet/espnet/pull/5184/commits/f83d811c4097c9c36e58e72ee7ceb5aa069af7c3). I'm checking

Reply:
Oh, I think you reviewed it before I uploaded the changed commit! thanks. Resolving.